### PR TITLE
FALCON-2063 Add change log for each release

### PR DESCRIPTION
**Pull request by ajayyadava**

### CODE REVIEW

File: release-docs/0.10/CHANGES.0.10.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+| [FALCON-1841](https://issues.apache.org/jira/browse/FALCON-1841) | Grouping test in falcon for running nightly regression |  Major | regression | Pragya Mittal | Pragya Mittal |
+| [FALCON-1835](https://issues.apache.org/jira/browse/FALCON-1835) | Falcon should do coord rerun rather than workflow rerun to ensure concurrency |  Major | . | Pallavi Rao | Pallavi Rao |
+| [FALCON-1818](https://issues.apache.org/jira/browse/FALCON-1818) | Minor doc update for tar package locations after FALCON-1751 |  Minor | . | Deepak Barr | Deepak Barr |
+| [FALCON-1774](https://issues.apache.org/jira/browse/FALCON-1774) | Better message for api not allowed on server |  Major | . | Sanjeev T | Praveen Adlakha |
+| [FALCON-1751](https://issues.apache.org/jira/browse/FALCON-1751) | Support assembly:single mojo |  Minor | . | ruoyu wang | ruoyu wang |
+| [FALCON-1584](https://issues.apache.org/jira/browse/FALCON-1584) | Falcon allows invalid hadoop queue name for schedulable feed entities |  Major | . | Venkatesan Ramachandran | Venkatesan Ramachandran |
+
+
+### BUG FIXES:
```
Comment (bvellanki) [inline release-docs/0.10/CHANGES.0.10.md]: Some important features are listed as bug fixes in the generated Change log. I will go ahead and update the 0.10 Jiras to properly reflect compatibility, improvement, feature etc in an Apache Yetus compatible way. Then we need to re-generate the change-log. 

The same needs to be done for all other releases or we lose some important information on backwards compatibility, features, improvements etc.

**Comment by bvellanki** [comment]
Some important new features are listed as bug fixes in the generated Change log. I will go ahead and update the 0.10 Jiras to properly reflect compatibility, improvement, feature etc in an Apache Yetus compatible way. Then we need to re-generate the change-log.

The same needs to be done for all other releases or we lose some important information on backwards compatibility, features, improvements etc. This is necessary because we look at changelog to debug any issues that arises during upgrades between versions.

**Comment by ajayyadava** [comment]
@bvellanki Agreed and it makes sense. Now we just have to keep JIRA correctly updated.  

This pull request was just to get a preview and get review done. I am also fixing some issues in the YETUS. Hopefully they will also be committed in the meantime  :smile: 

One more thing that we need to do is to add Release Notes in the JIRA itself so that we can generate release notes also automatically. I need to first get that field added in JIRA.

As for cleaning up the JIRA, I will help you out and pick up 0.7(I generated the release notes for 0.7 from JIRA so they should mostly be correct :stuck_out_tongue: ) and 0.8 releases.

**Comment by bvellanki** [comment]
I finished cleaning up the Jira for 0.10 and working on 0.9 now. Thanks @ajayyadava  for taking the initiative.

**Comment by bvellanki** [comment]
I finished cleaning up the Jira for 0.9 now. @ajayyadava , please regenerate the change log

**Comment by ajayyadava** [comment]
@bvellanki I think you may want to merge and fix [FALCON-2088](https://github.com/apache/falcon/pull/232) before this. Have a look and let me know.

Unfortunately, one of the changes that I proposed to the tool(and needed for falcon) is still not merged and hence I am unable to give you the command so that you can generate it as per your convenience. Ping/call me once you are done and I will generate the changes.

**Comment by bvellanki** [comment]
@ajayyadava  : I merged Falcon-2088

**Comment by ajayyadava** [comment]
@bvellanki I have generated change log for 0.10 only, since I haven't been able to verify the entire content of CHANGES.txt with the change log. This is just to unblock you for 0.10 RC.

**Comment by ajayyadava** [comment]
This needs one more change, I need to add it to packaging as well. I will put that as an additional change on top of these changes if you are satisfied with these changes. In the meantime I will test that patch :)

**Comment by bvellanki** [comment]
+1 for 0.10 changelog. Will commit

**Pull request closed without merging**